### PR TITLE
Minor fast transfers updates

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR makes some minor edits for consistency and formatting and also adds in the index page for the Fast Transfers section

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly

### CODE REVIEW

**Review by dawnkelly09 — DISMISSED** [review]
lgtm

File: build/contract-integrations/fast-transfers/index.md [hunk 1]
```diff
@@ -1,4 +1,50 @@
...
+
+<div class="grid cards" markdown>
+
+-   :octicons-arrow-switch-16:{ .lg .middle } **How to Be a Solver**
+
+    ---
+
+    Learn how to initiate and participate in cross-chain transfer auctions, submit and verify transactions, and settle auctions efficiently as a solver in the Fast Transfers protocol.
+
+    [:octicons-arrow-right-16: Learn how to be a solver](/build/contract-integrations/wormhole-relayers/)
```
Comment (ilariae) [inline build/contract-integrations/fast-transfers/index.md:18]:
```suggestion
    [:octicons-arrow-right-16: Learn how to be a solver](/build/contract-integrations/fast-transfers/how-to-solver/)
```

File: build/contract-integrations/fast-transfers/index.md [hunk 2]
```diff
@@ -1,4 +1,50 @@
...
+
+-   :octicons-file-code-16:{ .lg .middle } **Smart Contract Components**
+
+    ---
+
+    Take a look at the key components and processes behind the smart contracts that power the Fast Transfers protocol and how they facilitate efficient transfers across blockchain networks.
+
+    [:octicons-arrow-right-16: See the smart contract components](/build/contract-integrations/core-contracts/)
```
Comment (ilariae) [inline build/contract-integrations/fast-transfers/index.md:26]:
```suggestion
    [:octicons-arrow-right-16: See the smart contract components](/build/contract-integrations/fast-transfers/smart-components/)
```

**Review by ilariae — CHANGES_REQUESTED** [review]
lgtm just two links that need to be fixed

**Review by ilariae — APPROVED** [review]
lgtm!

**Pull request merged**